src/screens/SignUp/index.js

```diff
@@ -0,0 +1,102 @@
+import React, { useState } from 'react'
+import NavigationService from '../../navigationService'
+import { ImageBackground, Text, View, Image, TextInput, TouchableHighlight, ActivityIndicator, KeyboardAvoidingView } from 'react-native'
+import { Formik } from 'formik'
+import AppText from '../../helpers/AppText'
+import styles from './styles'
+import * as firebase from 'firebase'
+
+const SignUp = () => {
+  const [authenticating, setAuthenticating] = useState(false)
+  const [error, setError] = useState(null)
+
+  const handleSubmit = (values) => {
+    setAuthenticating(true)
+    firebase.auth().createUserWithEmailAndPassword(values.email, values.password)
+      .then(user => {
```

Comment:
Can we use `async/wait` instead of then/catch?

Reply:
Yes, it's possible